docs/learn-account-restore.md

```diff
@@ -62,6 +62,26 @@ your account has been restored. That's it!
 
 ![restore-complete-polkadot](assets/accounts/polkadot_restore_complete.png)
 
+### Using an existing Mnemonic Seed to backup an Account
+
+If you've already created an account but are looking to have it backed up, you can do that by using
+the Polkadot.js UI and the existing mnemonic seed.
+
+To start, you'll need to open up the [Polkadot.js UI](https://polkadot.js.org/apps) and head over to
+the accounts tab. Click on the "Add Account" button, and fill out with the prompted details: a name
+for the account, a password, and be sure to replace the generated mnemonic seed with your _existing
+seed_. Note here that when you replace the generated mnemonic seed with your existing one, you
+aren't actually creating a new account, rather adding that account onto the Polkadot.js UI.
```

Comment:
Perhaps make a note that the name and password of the account only apply to the restored copy and do not need to match the name and password originally defined, if any.

Reply:
Great point! I'll add that in